docs/en/installing/compilation-with-ldb-toolchain.md

```diff
@@ -0,0 +1,94 @@
+---
+{
+    "title": "Compiling with LDB toolchain",
+    "language": "en"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Compiling with LDB toolchain
+
+This document describes how to compile Doris using the LDB toolchain, which will later replace the Docker development image as the default source code compilation method for Doris.
+
+> Thanks to [Amos Bird](https://github.com/amosbird) for this contribution.
+
+## Prepare the environment
+
+This works for most Linux distributions (CentOS, Ubuntu, etc.).
+
+1. Download `ldb_toolchain_gen.sh`
+
+    The latest `ldb_toolchain_gen.sh` can be downloaded from [here](https://github.com/amosbird/ldb_toolchain_gen/releases). This script is used to generate the ldb toolchain.
+    
+    > For more information, you can visit [https://github.com/amosbird/ldb_toolchain_gen](https://github.com/amosbird/ldb_toolchain_gen)
+
+2. Execute the following command to generate the ldb toolchain
+
+    ```
+    sh ldb_toolchain_gen.sh /path/to/ldb_toolchain/
+    ```
+    
+    where `/path/to/ldb_toolchain/` is the directory where the toolchain is installed.
+    
+    After successful execution, the following directory structure will be created under `/path/to/ldb_toolchain/`.
+    
+    ```
+    ├── bin
+    ├── include
+    ├── lib
+    ├── share
+    ├── test
+    └── usr
+    ```
+    
+3. Download and install other compiled components
```

Comment:
Do we need to mention other dependencies? `ldb_toolchain` provides flex. However it's impossible to bundle autoconf, bison, etc. Here is a almost minimal centos 6 Dockerfile to compile Doris with `ldb_toolchain` https://gist.github.com/amosbird/113c8c06cb04ac2557af85350fe00c00

Reply:
I see. maybe we can just refer to your gist?
Or paste it in this document?

Reply:
I think we should avoid introducing the concept of docker. So it's better to paste the following directly into the documentation (`java-1.8.0-openjdk-devel` and `apache-maven` are ignored deliberately because you've mentioned them as dedicated blobs.):

```
The following instructions describe how to setup a minimal CentOS 6 box to compile Doris. Other linux distros should be similar.

# install required system packages
yum install -y byacc patch automake libtool make which file ncurses-devel gettext-devel unzip bzip2 zip util-linux wget git

# install autoconf-2.69
wget http://ftp.gnu.org/gnu/autoconf/autoconf-2.69.tar.gz && \
    tar zxf autoconf-2.69.tar.gz && \
    cd autoconf-2.69 && \
    ./configure && \
    make && \
    make install

# install bison-3.0.4
wget http://ftp.gnu.org/gnu/bison/bison-3.0.4.tar.gz && \
    tar xzf bison-3.0.4.tar.gz && \
    cd bison-3.0.4 && \
    ./configure && \
    make && \
    make install
```

Reply:
done